app/src/testDemo/kotlin/com/google/samples/apps/nowinandroid/ui/NiaAppScreenSizesScreenshotTests.kt

```diff
@@ -139,7 +140,9 @@ class NiaAppScreenSizesScreenshotTests {
             CompositionLocalProvider(
                 LocalInspectionMode provides true,
             ) {
-                TestHarness(size = DpSize(width, height)) {
+                DeviceConfigurationOverride(
```

Comment:
There seems to be behaviour change here. Previously, the screenshot would be the same size as the `BoxWithConstraints` but now it's fixed at 1500x1404px, top left aligned.

Reply:
Interesting, we are generating the screenshots from the root, and it looks like the root in `DeviceConfigurationOverride` is filling up the whole screen, instead of behaving like the one in testharness and more closely wrapping the content.

Reply:
Can you report a bug for `DeviceConfigurationOverride.ForcedSize`? Definitely agreed that `ForcedSize` shouldn't fill up the max space available if the requested size is smaller than that.

In the interim, we should be able to preserve behavior if we take a screenshot of the node specifically corresponding to the app, instead of the root.

Reply:
Bug filed here: https://b.corp.google.com/issues/327149166 and screenshot updated to use the app node instead of the root.